<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Andrey Cherepanov &lt;cas@altlinux.org&gt;</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cuneiform OCR</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainWindow.qrc">
    <normaloff>:/small/icons/cuneiform-qt.png</normaloff>:/small/icons/cuneiform-qt.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QDockWidget" name="imageWidget">
       <property name="whatsThis">
        <string>Source image</string>
       </property>
       <property name="features">
        <set>QDockWidget::NoDockWidgetFeatures</set>
       </property>
       <property name="windowTitle">
        <string>Source Image</string>
       </property>
       <widget class="QWidget" name="dockWidgetContents">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGraphicsView" name="source">
           <property name="whatsThis">
            <string>Source image</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="dragMode">
            <enum>QGraphicsView::ScrollHandDrag</enum>
           </property>
           <property name="transformationAnchor">
            <enum>QGraphicsView::NoAnchor</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QDockWidget" name="textWidget">
       <property name="features">
        <set>QDockWidget::NoDockWidgetFeatures</set>
       </property>
       <property name="windowTitle">
        <string>Recognized text</string>
       </property>
       <widget class="QWidget" name="dockWidgetContents_2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTextEdit" name="result">
           <property name="whatsThis">
            <string>Result text</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Verdana'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionRecognizeText"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionConfigure"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionRecognizeText"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="mainWindow.qrc">
     <normaloff>:/small/icons/small-open.png</normaloff>:/small/icons/small-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open image...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="mainWindow.qrc">
     <normaloff>:/small/icons/small-save.png</normaloff>:/small/icons/small-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save result...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Cuneiform-Qt</string>
   </property>
  </action>
  <action name="actionRecognizeText">
   <property name="icon">
    <iconset resource="mainWindow.qrc">
     <normaloff>:/small/icons/small-recognize.png</normaloff>:/small/icons/small-recognize.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Recognize text</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>&amp;Configure...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
